package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
ahh semver is so complex 😫 Thanks for pointing out a special treatment for 0.x.x releases. However why you are saying that typeorm still should follow 0.0.10 and not be 0.1.0 ? As I understood 0.1.0 will be `0.MAJOR.MINOR` which means api changes on MAJOR level are acceptable and on minor levels are not. This is equal to what I was talking:

> So for now strategy is to try to make breaking changes only between 0.x versions and all other non breaking changes in 0.0.x versions.